openmc/mgxs/__init__.py

```diff
@@ -6,12 +6,18 @@
 from openmc.mgxs.mdgxs import *
 
 GROUP_STRUCTURES = {}
-"""Dictionary of commonly used energy group structures, including "CASMO-X" (where X
-is 2, 4, 8, 16, 25, 40 or 70) from the CASMO_ lattice physics code and other commonly
-used activation_ energy group structures "VITAMIN-J-175", "TRIPOLI-315", "CCFE-709_" 
-and "UKAEA-1102_"
+"""Dictionary of commonly used energy group structures:
+- "CASMO-X" (where X is 2, 4, 8, 16, 25, 40 or 70) from the CASMO_ lattice
+physics code
+- "XMAS-172" designed for LWR analysis (Sartori 1990, Santamarina et al. 2004)
```

Comment:
These references are not listed anywhere. I'd prefer hyperlinks using DOI numbers: `https://doi.org/...`

Reply:
To my knowledge after looking for a few hours, none of this has DOI numbers

Santamarina 2004 is a PHYSOR paper (https://inis.iaea.org/search/search.aspx?orig_q=RN:42097652)
Sartori 1990 is : 
Sartori E OECD/NEA Data Bank, Standard energy group structures of cross section libraries for reactor shielding, reactor cell and fusion neutronics applications: VITAMIN-J, ECCO-33, ECCO2000 and XMAS, JEF/DOC-315 Revision 3
RSICC distributes it if anyone wants the CD : [rsicc](https://rsicc.ornl.gov/codes/dlc/dlc2/dlc-274.html)

Hfaiedh 2005 is an M&C paper
Hebert 2008 is a physor paper

Reply:
My point is that the references (DOI or not) need to be included in the docstring or else the users have nothing to refer to. In the case that there's no good hyperlink, I would use a restructured text [citation](http://docutils.sourceforge.net/docs/user/rst/quickref.html#citations). Also, you added a `_` after Sartori but there is no corresponding hyperlink.

Reply:
I went for citations. A Google search of the title will bring up the article.